vision/unit_tests/test_client.py

```diff
@@ -15,16 +15,16 @@
 
 import unittest
 
-from google.cloud._helpers import _to_bytes
-
-_IMAGE_CONTENT = _to_bytes('/9j/4QNURXhpZgAASUkq')
-_IMAGE_SOURCE = 'gs://some/image.jpg'
-
 
 class TestClient(unittest.TestCase):
     import base64
+    from google.cloud._helpers import _to_bytes
+    from google.cloud._helpers import _bytes_to_unicode
```

Comment:
To build the results.
```python
    import base64
    from google.cloud._helpers import _to_bytes
    from google.cloud._helpers import _bytes_to_unicode

    _IMAGE_CONTENT = _to_bytes('/9j/4QNURXhpZgAASUkq')
    _IMAGE_SOURCE = 'gs://some/image.jpg'
    PROJECT = 'PROJECT'
    B64_IMAGE_CONTENT = _bytes_to_unicode(base64.b64encode(_IMAGE_CONTENT))
```

Should these be in the module scope in this case or just import them in each test?